lib/ansiblelint/__main__.py

```diff
@@ -31,18 +32,19 @@
 
 
 def main():
+    cwd = os.getcwd()
 
-    formatter = formatters.Formatter()
     options, args = cli.get_config(sys.argv[1:])
 
+    formatter = formatters.Formatter(cwd, options.relative_path)
     if options.quiet:
-        formatter = formatters.QuietFormatter()
+        formatter = formatters.QuietFormatter(cwd, options.relative_path)
 
     if options.parseable:
-        formatter = formatters.ParseableFormatter()
+        formatter = formatters.ParseableFormatter(cwd, options.relative_path)
 
     if options.parseable_severity:
-        formatter = formatters.ParseableSeverityFormatter()
+        formatter = formatters.ParseableSeverityFormatter(cwd, options.relative_path)
```

Comment:
Maybe call it in one place?
```suggestion
    formatter_factory = formatters.Formatter

    if options.quiet:
        formatter_factory = formatters.QuietFormatter

    if options.parseable:
        formatter_factory = formatters.ParseableFormatter

    if options.parseable_severity:
        formatter_factory = formatters.ParseableSeverityFormatter

    formatter = formatter_factory(cwd, options.relative_path)
```